update document of tidb-ctl (#3048)
pingcap/docs: tidb-control.md

# TiDB Control User Guide

TiDB Control is a command-line tool of TiDB, usually used to obtain the status information of TiDB for debugging. This document introduces the features of TiDB Control and how to use these features.

## Get TiDB Control

> You can get TiDB Control using the following two methods.

Reviewer: ```suggestion
You can get TiDB Control by installing it using TiUP or by compiling it from source code.
```

- `-n`: the option
- `db`: the flag of `-n`

> Currently, TiDB Control contains following commands:

Reviewer: ```suggestion
Currently, TiDB Control has the following subcommands:
```

> - `tidb-ctl base64decode`: BASE64 decode

Reviewer: ```suggestion
- `tidb-ctl base64decode`: used for `BASE64` decoding
```

> - `tidb-ctl decoder`: for KEY decode

Reviewer: language and format ditto

> - `tidb-ctl etcd`: for operating etcd

Reviewer: ```suggestion
- `tidb-ctl etcd`: used for operating etcd
```

> - `tidb-ctl log`: format the log file to expand the single-line stack information

Reviewer: ```suggestion
- `tidb-ctl log`: used to format the log file to expand the single-line stack information
```

> - `tidb-ctl mvcc`: used to get the mvcc information

Reviewer: ```suggestion
- `tidb-ctl mvcc`: used to get the MVCC information
```

> - `tidb-ctl region`: used to get the region information

Reviewer: ```suggestion
- `tidb-ctl region`: used to get the Region information
```

- `tidb-ctl schema`: Schema information

> - `tidb-ctl table`: Table information

Reviewer: used to get the ...

### Get help

Use `tidb-ctl -h/--help` to get usage information.

TiDB Control consists of multiple layers of commands. You can use `-h/--help` after each command/subcommand to get its respective usage information.

The following example shows how to obtain the schema information:

Use `tidb-ctl schema -h` to get usage details. The `schema` command itself has two subcommands: `in` and `tid`.

- `in` is used to obtain the table schema of all tables in the database through the database name.
- `tid` is used to obtain the table schema by using the unique `table_id` in the whole database.

### Global options

> `tidb-ctl` has 7 connection related global options:

Reviewer: ```suggestion
`tidb-ctl` has the following connection-related global options:
```

**Connection options are global options that apply to all the following commands:**

- `tidb-ctl base64decode`
- `tidb-ctl decoder`
- `tidb-ctl etcd`
- `tidb-ctl log`
- `tidb-ctl mvcc`
- `tidb-ctl region`
- `tidb-ctl schema`

> - `tidb-ctl table`

Reviewer: I think you can remove this part.

> ### The `schema` connand

Reviewer: ```suggestion
### The `schema` command
```

#### The `in` subcommand

> The `in` subcommand is used to get table information from target database.

Reviewer: ```suggestion
`in` is used to obtain the table schema of all tables in the database through the database name.
```

    If you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure. For example, `tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`.

#### The `tid` subcommand

> - `tid` is used to obtain the table schema by using the unique `table_id` in the whole database. you can use `in` subcommand to get all table id of certain schema and use `tid` subcommand to get detail table information.

Reviewer: ```suggestion
`tid` is used to obtain the table schema by using the unique `table_id` in the whole database. You can use the `in` subcommand to get all table IDs of certain schema and use the `tid` subcommand to get the detailed table information.
```

    If you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure. For example, `tidb-ctl --host 172.16.55.88 --port 8898 schema in mysql -n db`.

#### The `tid` subcommand

> The `tid` subcommand is used to get table information and structure by table id.

Reviewer: ```suggestion
- `tid` is used to obtain the table schema by using the unique `table_id` in the whole database.
```

> 

Reviewer: `通过使用 in 子命令查询到数据库中表的 id，之后可以通过 tid 子命令查看表的详细信息。` is missing from your translation.

> For example, the table id of `mysql.stat_meta` is 21, you can use `tidb-ctl schema tid -i 21` to obtain the detail of `mysql.stat_meta`.

Reviewer: ```suggestion
For example, the table ID of `mysql.stat_meta` is `21`. You can use `tidb-ctl schema tid -i 21` to obtain the detail of `mysql.stat_meta`.
```

```json
{
 "id": 21,
 "name": {
  "O": "stats_meta",
  "L": "stats_meta"
 },
 "charset": "utf8mb4",
 "collate": "utf8mb4_bin",
  ...
}
``` 

> Like the `in` subcommand, if you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to configure.

Reviewer: ```suggestion
Like the `in` subcommand, if you do not want to use the default TiDB service address and port, use the `--host` and `--port` options to specify the host and port.
```

> #### The `base64decode` subcommand

Reviewer: ```suggestion
#### The `base64decode` command
```
